appmap/pytest.py

```diff
@@ -4,15 +4,15 @@
 from appmap._implementation import testing_framework
 
 
-@pytest.hookimpl
-def pytest_runtestloop(session):
-    # Use the same session for unittest in case there are any unittest.TestCases.
-    appmap.unittest.session = session.appmap = testing_framework.session(name='pytest', version=pytest.__version__)
+if appmap.enabled():
```

Comment:
@apotterri Please review this specifically.

I added a pytest fixture to set APPMAP environment variable so I don't have to repeat it every time. But the pytest hooks end up interfering somehow (haven't had the bandwidth to investigate in detail).

Adding the hooks only when the environment is enabled beforehand seems to handle it properly.

All tests are passing.

Reply:
@virajkanwade You mentioned that all your tests are passing. I don't see any tests for the flask integration, though I see the files that would support them.

Do you have a local file with tests in it?

Reply:
Aah, can you check now.